### PR TITLE
Multi file, multi folder support

### PR DESCRIPTION
**Pull request by helmecke**
*Issue #47 , if available:*

*Description of changes:*
Added support for multiple files in multiple folders.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by knqyf263** [comment]
Thank you for the awesome contribution! Let me know why you removed config.go. Also, it would be appreciated if you show an example to use this new feature.

**Comment by helmecke** [comment]
Thanks and sorry for the late reply.

> Let me know why you removed config.go.
`Viper` is very intuetive to work with and integrates nicely with `Cobra`. It removes the need to maintain an additional file which changes every time the config is changed.

> Also, it would be appreciated if you show an example to use this new feature.
I will add a section to the readme as soon as possible.

**Comment by helmecke** [comment]
@knqyf263 I've added the Readme section. I would very much like your feedback.

**Comment by brandonkal** [comment]
There's a typo in the readme change.

**Comment by knqyf263** [comment]
I'm so sorry to be too late. I'm reviewing this PR.

File: Makefile [hunk 1]
```diff
@@ -1,56 +1,18 @@
...
-	update \
-	build \
-	install \
-	lint \
-	vet \
-	fmt \
-	fmtcheck \
-	clean \
-	pretest \
-	test
```
Comment (knqyf263) [inline Makefile:14]: Is there any reason to remove them?

File: cmd/clip.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	"strings"
+
+	"github.com/atotto/clipboard"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// clipCmd represents the clip command
+var clipCmd = &cobra.Command{
```
Comment (knqyf263) [inline cmd/clip.go:14]: This feature is not related to multi-folder support, right? If so, could you open another PR for this feature?

File: cmd/completion.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+var (
+	completionShells = map[string]func(out io.Writer, cmd *cobra.Command) error{
+		"bash": runCompletionBash,
+		"zsh":  runCompletionZsh,
+	}
+)
+
+// completionCmd represents the completion command
+var completionCmd = &cobra.Command{
```
Comment (knqyf263) [inline cmd/completion.go:19]: ditto

**Review by knqyf263 — CHANGES_REQUESTED** [review]
Could you revert config.go? If we don't have it, we have to access each value with a string like `viper.GetString("gitlab.file_name")` everywhere as you did. This is likely to cause typos and is not desirable for completion. If you strongly hope it, let's open the dedicated issue and discuss it.

**Comment by helmecke** [comment]
@knqyf263 it was my bad to open this PR from master branch. I'll try to cleanup this PR to only include multi-folder and multi-file support.

**Comment by Piotr1215** [comment]
Hi everyone. Thank you for this great project! Is there any timeline on when this PR will be merged? I'm using @helmecke fork for now.

**Pull request closed without merging**